Sources/Document/Json/JSONArray.swift

```diff
@@ -381,7 +381,7 @@ public class JSONArray {
     @discardableResult
     private func removeInternal(byIndex index: Int) -> CRDTElement? {
         let ticket = self.context.issueTimeTicket
-        let removed = try? self.target.remove(index: index, executedAt: ticket)
+        let removed = try? self.target.deleteByIndex(index: index, executedAt: ticket)
```

Comment:
Using `guard` will keep the code shorter

```swift        
guard let removed = try? self.target.deleteByIndex(index: index, executedAt: ticket) else {
     return nil
}
```

Reply:
Thanks.